base/show.jl

```diff
@@ -1213,10 +1213,11 @@ function show_unquoted(io::IO, ex::Expr, indent::Int, prec::Int)
         # unary operator (i.e. "!z")
         elseif isa(func,Symbol) && func in uni_ops && length(func_args) == 1
             show_unquoted(io, func, indent)
-            if isa(func_args[1], Expr) || isoperator(func_args[1])
+            arg1 = func_args[1]
+            if isa(arg1, Expr) || (isa(arg1, Symbol) && isoperator(arg1))
```

Comment:
It seems like several of these calls (e.g. also in #26721) would be simplified if we defined `isoperator(x) = false`.